bff-service/BffService/Startup.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Microsoft.Extensions.Caching.Memory;
+
+namespace BffService;
+
+public class Startup
+{
+    public Startup(IConfiguration configuration)
+    {
+        Configuration = configuration;
+    }
+
+    private readonly Dictionary<string, string> _routes = new();
+
+    private void LoadRoutes()
+    {
+        var productService = Environment.GetEnvironmentVariable("PRODUCT_SERVICE");
+        if (!string.IsNullOrWhiteSpace(productService))
+            _routes.Add("product-svc", productService);
```

Comment:
with such an approach you'll need to **modify source code** and **redeploy** bff-service each time, when a new route is added.

Read again the requirements in Task 10.1:
> 3. Here's the workflow example that BFF Service should support:
> * Make requests to BFF Service with URL in the following format: `{bff-service-url}/{recipient-service-name}?var1=someValue`
> 
> * Get `recipientURL` from the env variables using `{recipient-service-name}` as a key.

the idea is to simplify maintenance. So when we want to add a new route `/new-svc`, we will just add a new environment variable like:
```
new-svc=https://new-url.com
```